CRM/Odoosync/Sync/Contribution.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * Handles syncing contribution data to odoo
+ */
+class CRM_Odoosync_Sync_Contribution extends CRM_Odoosync_Sync_BaseHandler {
+
+  /**
+   * The contribution ID to sync
+   *
+   * @var int
+   */
+  private $syncContributionId;
+
+  /**
+   * Starts contribution Odoo sync
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  protected function startSync() {
+    $this->setLog(ts('Start Contribution Syncing ...'));
+    $this->setJobLog(ts('Start Contribution Syncing ...'));
+
+    $pendingContribution = new CRM_Odoosync_Sync_Contribution_PendingContribution();
+    $contributionIdList = $pendingContribution->getPendingContributions();
```

Comment:
why to repeat the class name in the method name, changing the class name to  CRM_Odoosync_Sync_Contribution_PendingContribution**s** and the methods name to getIds() would be enough .